src/state/item/item-action-creators.js

```diff
@@ -92,7 +78,7 @@ export function saveMetadata() {
         itemApi.updateItemMetadata(activeItem, activeItemEditing)
             .then((success) => {
                 dispatch({ type: itemActionTypes.METADATA_SAVE_SUCCEEDED });
-                dispatch(fetchItems(currentPage));
+                dispatch(fetchItem(activeItem.id));
```

Comment:
Didn't see a reason to refresh all the documents in the list when we could just refresh the item that was updated.